duckduckgo/Android
# Update tracker radar url and embedded data

Task/Issue URL:  https://app.asana.com/0/361428290920652/1175573289211364/f

**Description**:
Update to latest embedded tracker data set and url/

**Steps to test this PR**:

Test the following on a high-end and low-end device

**Test update url:**
1. Update tds.json to be empty `{}`
1. Run a fresh version of the app and wait for download sync job to complete, you will see "Download task completed successfully" in the logs
1. Visit a website e.g cnn.com and ensure that trackers are blocked

**Test embedded when download fails:**
1. Revert previous changes
1. Update the the tds.json link in `TrackerListService` to be giberish so the download will failr
1. Install the app
1. Visit a website e.g cnn.com and ensure that trackers are blocked

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
